app/controllers/v0/decision_review_evidences_controller.rb

```diff
@@ -15,38 +15,39 @@ class DecisionReviewEvidencesController < ApplicationController
     private
 
     # This method, declared in `FormAttachmentCreate`, is responsible for uploading file data to S3.
-    def save_attachment_to_cloud! # rubocop:disable Metrics/MethodLength
+    def save_attachment_to_cloud!
       # `form_attachment` is declared in `FormAttachmentCreate`, included above.
       form_attachment_guid = form_attachment&.guid
       password = filtered_params[:password]
 
-      save_validation_data(form_attachment_guid:, password:)
-
-      common_log_params = {
-        key: :evidence_upload_to_s3,
-        form_id: get_form_id_from_request_headers,
-        user_uuid: current_user.uuid,
-        downstream_system: 'AWS S3',
-        params: {
-          form_attachment_guid:,
-          encrypted: password.present?
-        }
+      params = {
```

Comment:
nit picky, but can we rename this to log_params? — when i see params in a controller i think of the [params hash from the request](https://guides.rubyonrails.org/action_controller_overview.html#parameters)

Reply:
Done 👍